.github/PULL_REQUEST_TEMPLATE/Documentation.md

```diff
@@ -0,0 +1,23 @@
+---
+name: 📚 Documentation
```

Comment:
We don't have any documentation in this repository.
For Parcel 2 we'll have typings and a spec but the typings should be changed whenever the code changes, so it's never seperate. (Hopefully) So I'm not sure docs should have a PR template, spec changes will rarely happen.

However it would be awesome if we would have a template on both issues and PRs for docs that just says, this is not the docs repo file an issue/PR here: https://github.com/parcel-bundler/website

Reply:
Sure that makes sense. I was just adapting the issue templates and made them fit to PRs.
You're free to edit them if you want, but I can do also when I'm back at work tomorrow.